__tests__/components/editor/property/InputLiteral.test.js

```diff
@@ -71,20 +71,25 @@ describe('checkMandatoryRepeatable', () => {
 })
 
 describe('When the user enters input into field', () => {
-  // Our mock formData function to replace the one provided by mapDispatchToProps
-  const mockFormDataFn = jest.fn()
+  // Our mockItemsChange function to replace the one provided by mapDispatchToProps
+  let mockItemsChange
   const removeMockDataFn = jest.fn()
 
   shortid.generate = jest.fn().mockReturnValue(0)
-  const mockWrapper = shallow(<InputLiteral {...plProps} id={'11'}
-                                            rtId={'resourceTemplate:bf2:Monograph:Instance'}
-                                            reduxPath={[
-                                              'resourceTemplate:bf2:Monograph:Instance',
-                                              'http://id.loc.gov/ontologies/bibframe/instanceOf',
-                                            ]}
-                                            handleMyItemsChange={mockFormDataFn}
-                                            handleRemoveItem={removeMockDataFn}
-                                            handleMyItemsLangChange={jest.fn()} />)
+  let mockWrapper
+
+  beforeEach(() => {
```

Comment:
Like this refactoring; setting `mockItemsChange` to the jest function mock in the beforeEach makes sense.